src/wwwroot/controllers/BillingCtrl.js

```diff
@@ -153,6 +154,7 @@
     }
 
     var onExpectedError = function (rejectionData) {
+      vm.processingPayment = false;
```

Comment:
What do you think about using _finally_ (or something like that)

Reply:
User is redirected to My Plan on success, so it's not strictly necessary but finally looks cleaner and self-descriptive. I'll change it.